core_impl/src/main/java/com/google/instrumentation/stats/MeasurementDescriptorToViewMap.java

```diff
@@ -52,4 +53,23 @@ final View getView(ViewDescriptor viewDescriptor) {
   void putView(MeasurementDescriptor.Name name, View view) {
     mutableMap.put(name, view);
   }
+
+  // Records stats with a set of tags.
+  void record(Map<String, String> tags, MeasurementMap stats) {
+    for (MeasurementValue mv : stats) {
+      if (mv.getMeasurement()
```

Comment:
Could we check for supported measure descriptor before we enqueue a new `StatsEvent` (instead of checking here)?